cosmos/operators/local.py

```diff
@@ -317,28 +314,22 @@ def _construct_dest_file_path(
         dag_task_group_identifier = self.extra_context["dbt_dag_task_group_identifier"]
         rel_path = os.path.relpath(file_path, source_compiled_dir).lstrip("/")
 
-        return f"{dest_target_dir_str}/{dag_task_group_identifier}/compiled/{rel_path}"
-
-    def upload_compiled_sql(self, tmp_project_dir: str, context: Context) -> None:
-        """
-        Uploads the compiled SQL files from the dbt compile output to the remote store.
-        """
-        if not self.should_upload_compiled_sql:
-            return
+        return f"{dest_target_dir_str}/{dag_task_group_identifier}/{resource_type}/{rel_path}"
 
+    def _upload_sql_files(self, tmp_project_dir: str, resource_type: str) -> None:
         dest_target_dir, dest_conn_id = self._configure_remote_target_path()
 
         if not dest_target_dir:
             raise CosmosValueError(
-                "You're trying to upload compiled SQL files, but the remote target path is not configured. "
+                "You're trying to upload run SQL files, but the remote target path is not configured. "
```

Comment:
```suggestion
                "You're trying to upload SQL files, but the remote target path is not configured. "
```
Why would we specify "run"?